common/production_methods/06_urban_center.txt

```diff
@@ -760,9 +752,6 @@ pm_enhanced_geothermal_systems = {
 pm_nuclear_plant = {
 	texture = "gfx/interface/icons/production_method_icons/power_plant.dds"
 	pollution_generation = 10
-	unlocking_technologies = {
-		nuclear_fission
```

Comment:
surely this shouldn't be allowed at game start.

Reply:
Since its the first PM for the PMG, it shouldn't have a technology requirement.  The tech requirement is on the building itself, not the PM.

Reply:
copy that 🫡